requirements.txt

```
@@ -1,6 +1,6 @@
 pyiron==0.5.2
-pyiron_base==0.10.10
-pyiron-atomistics==0.6.19
+pyiron_base==0.11.1
+pyiron-atomistics==0.6.23
```

It is interesting that `pyiron_base` is written with `_` and `pyiron-atomistics` with `-`. Maybe this is related to the issue that dependabot is struggling to update them. At least on pypi they are both listed with an `-`:
* https://pypi.org/project/pyiron-atomistics/
* https://pypi.org/project/pyiron-base/

---

O.o very good and important catch!
```suggestion
pyiron-base==0.11.1
pyiron-atomistics==0.6.23
```